api/logics.py

```diff
@@ -736,9 +736,10 @@ def payout_amount(cls, order, user):
             valid = cls.create_onchain_payment(
                 order, user, preliminary_amount=context["invoice_amount"]
             )
-            order.log(
-                f"Suggested mining fee is {order.payout_tx.suggested_mining_fee_rate} Sats/vbyte, the swap fee rate is {order.payout_tx.swap_fee_rate}%"
-            )
+            if order.payout_tx is not None:
```

Comment:
`order.payout_tx` should not be None. If it is, the order is in a bad/corrupted in some way and the client will receive 500s. This could happen when LND has lower than 300K Sats onchain (node reserve). Are you testing with CLN?

Reply:
Ok after the cln update this problem seems to have been solved, I have removed this change from the PR.